packages/forms-web-app/__tests__/unit/controllers/examination/has-interested-party-number.test.js

```diff
@@ -156,16 +142,16 @@ describe('controllers/examination/has-interested-party-number', () => {
 			expect(res.redirect).toHaveBeenCalledWith('/examination/your-interested-party-number');
 		});
 
-		it('should redirect to /examination/your-interested-party-number', () => {
+		it('should redirect to /examination/are-you-applicant', () => {
 			const mockRequest = {
 				...req,
 				body: {
-					['examination-has-interested-party-number']: 'no'
+					'examination-has-interested-party-number': 'no'
 				}
 			};
 
 			postHasInterestedPartyNumber(mockRequest, res);
-			expect(res.redirect).toHaveBeenCalledWith(`${examinationDirectory + applicantRoute}`);
+			expect(res.redirect).toHaveBeenCalledWith('/examination/are-you-applicant');
```

Comment:
Not better to keep the config route so when they get changed the test still works? I know this can be a downside in case it gets set to an improper path but it should be noticed on the review or at latest on dev.

Reply:
@mauriciokin Reasoning for hardcoding values in test is to avoid unintentional changes in the config as any changes will be flagged in failing tests.